ft_hw:
  type: cnr/control/RobotiqForceTorqueHw
  appenders: [file, screen]
  levels: [trace, info]
  pattern_layout: "[%5p][%d{HH:mm:ss,SSS}][%M:%L][%c] %m%n"
  file_name: "ft_hw"
  default_throttle_time: 20.0
  sampling_period: 0.01
  init_sensor: true

  controlled_resource: robotiq_ft_sensor


  remap_source_args:
  - "/ft_hw/simple_touch/target_cart_twist"
  - "/ft_hw/robotiq_ft_sensor"

  remap_target_args:
    - "/target_cart_twist"
    - "/robotiq_ft_wrench"

  ft_pub:
    type: force_torque_sensor_controller/ForceTorqueSensorController
    publish_rate: 100

  simple_touch:
      type: cnr/control/SimpleTouchController
      sensor_name : robotiq_ft_sensor
      base_link   : base_link
      tool_link   : tool0
      sensor_frame: robotiq_ft_frame_id

